scripts/services/dependencies.ts

```diff
@@ -59,6 +59,7 @@ const proxyChecks: Array<Check> = [
   },
   ...[
     "estimatesmartfee",
+    "getblock",
```

Comment:
When is this needed?

Reply:
I'm not sure precisely but it is definitely being called by electrs at times, according to the btc-rpc-proxy logs, and when enabled, their (mostly sparrow) problems abated.